### PR TITLE
PMM-8209 MongoDB helper

### PR DESCRIPTION
**Pull request by yurkovychv**

### CODE REVIEW

File: pr.codecept.js [hunk 1]
```diff
@@ -25,6 +25,13 @@ exports.config = {
         ],
       },
     },
+    MongoDB: {
+      require: './tests/helper/mongoDB.js',
+      host: '127.0.0.1',
+      port: 27017,
+      username: 'root',
+      password: 'root-password',
```
Comment (puneet0191) [inline pr.codecept.js:33]: please change this according to the compose mongo details from my PR

**Review by puneet0191 — DISMISSED** [review]

File: pr.codecept.js (hunk 1 above)
Comment (yurkovychv) [reply pr.codecept.js]: done, thanks

File: tests/helper/mongoDB.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+   * Disconnects from mongo shell
+   * @returns {Promise<void>}
+   */
+  async mongoDisconnect() {
+    await this.client.close();
+  }
+
+  /**
+   * Runs a command against the current database
+   * readmore here: https://docs.mongodb.com/manual/reference/command/
```
Comment (atymchuk) [inline tests/helper/mongoDB.js:36]:
```suggestion
   * read more here: https://docs.mongodb.com/manual/reference/command/
```

File: tests/helper/mongoDB.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
+const { MongoClient } = require('mongodb');
+
+class MongoDB extends Helper {
```
Comment (atymchuk) [inline tests/helper/mongoDB.js:3]: By the name, it's a connection to the daemon, which it is not.
If it interfaces with the shell, should we give it a more explicit name, i.e. `class MongoDBShell`?
Up to you anyway )

**Review by atymchuk — APPROVED** [review]

File: tests/helper/mongoDB.js (hunk 3 above)
Comment (yurkovychv) [reply tests/helper/mongoDB.js]: Thanks I've changed it to `MongoDBHelper`

File: tests/helper/mongoDB.js (hunk 2 above)
Comment (yurkovychv) [reply tests/helper/mongoDB.js]: done, thanks

**Review by puneet0191 — APPROVED** [review]

**Review by IvanaBrecherova — APPROVED** [review]

**Pull request merged**